### ISSUE TITLE
The gradle task is not being picked up

### ISSUE THREAD

**Issue by bogdanzurac** [stoicflame/enunciate-gradle#3]
I'm trying to make the enunciate gradle task work.

```
tasks.enunciate {
    configFileName = "enunciate.xml"
    File enunciateDistDir = file("src/main/webapp")
    include("**/*.java")
    doFirst {
        enunciateDistDir.deleteDir()
        enunciateDistDir.mkdirs()
    }
    export("docs", enunciateDistDir)
}
```

Throws out:

```
Error:(34, 0) Build script error, unsupported Gradle DSL method found: 'enunciate()'!
```

If I try to use the below method instead (only first line is changed):

```
task buildDocs(type: com.webcohesion.enunciate.EnunciateTask) {
    configFileName = "enunciate.xml"
    File enunciateDistDir = file("src/main/webapp")
    include("**/*.java")
    doFirst {
        enunciateDistDir.deleteDir()
        enunciateDistDir.mkdirs()
    }
    export("docs", enunciateDistDir)
}
```

It throws:

```
Error:(34, 0) Cannot create task of type 'EnunciateTask' as it does not implement the Task interface.
```

So what's wrong?

**Comment by jskov-jyskebank-dk** [comment]
Did you apply the plugin?
Which version of Gradle are you using?

**Comment by bogdanzurac** [comment]
```
plugins {
    id "com.webcohesion.enunciate" version "2.1.1"
}

apply plugin: 'war'
apply plugin: 'com.webcohesion.enunciate'
```

Gradle v2.10. I can even access the EnunciateTask source code, so the plugin is there.

**Comment by jskov-jyskebank-dk** [comment]
Did the example on the front page work as is?
Maybe my example got broken...

I don't have time to look closer at it right now. Maybe later this evening.

**Comment by bogdanzurac** [comment]
Well the example just includes the plugin and configures the enunciate task. As I said, the plugin integration seems to be working just fine. The attempt to configure the task breaks, no matter how you want to configure it. So no, the example on the front page doesn't work as well.

**Comment by bogdanzurac** [comment]
As I haven't followed the way a task is defined in Gradle for the past n iterations of their releases (I didn't have a need for this on Android, but I do so on web currently), I can't vouch for my explanation of this issue; but I presume the way of defining a task has recently changed, from what your example shows vs how it should be done currently (as I tried above). So I would suggest taking a look at this as the primary cause of why it isn't working anymore. Dunno what needs to be changed though.

**Comment by jskov-jyskebank-dk** [comment]
It appears that the construct:

```
plugins {
    id "com.webcohesion.enunciate" version "2.1.1"
}
```

does not work.

Use the other construct instead:

```
buildscript {
...
}
```

**Comment by bogdanzurac** [comment]
Yes, it's working with the old method. Any idea why it isn't working with the plugins method though?

**Comment by jskov-jyskebank-dk** [comment]
Yes, documented (and fixed) in #4 
So it should work from next release (not sure when @stoicflame plans it)

**Comment by bogdanzurac** [comment]
Ok cool, thanks. Will check it out next time I'll be fiddling around with the deps.
